README.md

```diff
@@ -27,6 +27,7 @@ BIDS currently supports the following data modalities with more to come in the f
 - physiological
 - PET
 - microscopy
+- fNIRS
```

Comment:
NIRS is the method, but it has much broader uses such as food and atmospheric chemistry (e.g. scanning fruit for quality issues). When applied to neuroimaging we use fNIRS. So I think this should remain fNIRS. I also re-read the main md file, and we consistently use fNIRS (and I think this sounds correct). The coding and validator uses nirs. Is this a big issue? If it is a big issue, and we want to remove the "functional" from everywhere, I would want to get approval from @lpollonini first.

Reply:
I think I'd prefer it to be consistently fNIRS then ... not sure if worth the effort though :thinking: WDYT @rwblair @effigies @tsalo @Remi-Gau ?

Reply:
AS @rob-luke said, NIRS has a variety of non-functional applications, and the SNIRF extension (which somehow guided the design of this BEP) was crafted with a wide scope to keep those in mind.
Practically, I not clearly see this BEP being widely adopted outside of neuroimaging, i.e. functional NIRS., so I am in favor of using the narrower, yet best-fitting "fNIRS" everywhere.

Reply:
> not sure if worth the effort though 

This echoes my opinion. And we do use a NIRS measurement to perform fNIRS neuroimaging, so I don't think anything is technically incorrect. I am eager to hear the opinions of others on this, thanks for the detailed review!

Reply:
I am also in favor of using NIRS instead of fNIRS. There are for example also researchers that use NIRS to study orthostatic hypotension in the brain. They use the same devices as we do, so could benefit from the BEP, but are technically not interested in the 'functional' part of fNIRS. (see [this study](https://www.artinis.com/blogpost-all/2022/the-mhealth-project-investigating-orthostatic-hypotension-in-elderly-performing-long-term-nirs-measurements-in-a-home-based-setting) for example)